conpr/08_LE_Tasks/src/le/fj/Sum.java

```diff
@@ -26,27 +30,34 @@ protected Integer compute() {
             return sum; 
         } else {
             int mid = (hi + lo) / 2;
-            Sum firstHalf = new Sum(array, lo, mid);
-            firstHalf.fork();
-            Sum secondHalf = new Sum(array, mid, hi);
-            return secondHalf.compute() + firstHalf.join();
+            Sum firstHalf = new Sum(array, lo, mid, exec);
+            Sum secondHalf = new Sum(array, mid, hi, exec);
+            try {
+                return exec.submit(secondHalf).get() + exec.submit(firstHalf).get();
+            } catch (Exception e) {
+                e.printStackTrace();
+                return 0;
+            }
         }
     }
     
     public static void main(String[] args) {
         Random rnd = new Random();
         int SIZE = 40000000;
-        ForkJoinPool fj = new ForkJoinPool();
-      
+        ExecutorService fj = Executors.newVirtualThreadPerTaskExecutor(); 
```

Comment:
Description of driver method:
Here, they populate an array of size SIZE (originally 500 million; I changed this number to 40 million in [this PR](https://github.com/ponder-lab/FHNW-Java-Projekte/pull/2/files#diff-62aa5cd46958d4cbbc198e41583fdc44ce9c7e33dfd7b9505e0fa52a951fc612R38)) with random integers from 0 to 99, and compute the sum of its elements using a recursive tree of Sum tasks.